test/integration_test.js

```diff
@@ -22,7 +22,11 @@ describe("Integration tests", () => {
   var sandbox, server, tasks;
   const MAX_ATTEMPTS = 50;
 
-  function startServer(env={}) {
+  function startServer(env) {
```

Comment:
What if I call `startServer([])`? Defaulting a required object to an object sounds fair to me.